src/workerd/jsg/jsg.c++

```diff
@@ -194,12 +194,14 @@ void Lock::setLoggerCallback(kj::Function<Logger>&& logger) {
 }
 
 void Lock::requestGcForTesting() const {
+#ifdef KJ_DEBUG
```

Comment:
It'd be better to do this in `WorkerEntrypoint::maybeAddGcPassForTest()`. Otherwise, there's still a bunch of extra work that happens there in predictable mode, in particular taking an isolate lock, which will slow down load tests for no reason.

Reply:
Done